### DISCUSSION TITLE
posts/routers-on-cloudflare-pages/

### DISCUSSION THREAD

**Discussion by giscus** [yunnysunny/yunnysunny.github.io#28] [Show and tell]
# posts/routers-on-cloudflare-pages/

在之前的文章中，谈到了如何初始化 cloudflare pages。这篇文章将更输入的展开，讲解如何使用 pages 中的路由功能，和如何在实际中进行应用。

https://blog.whyun.com/posts/routers-on-cloudflare-pages/

**Comment by mechille** [comment]
文章写得很好很详细，期待后续更新
